node/src/tasks/bsp_upload_file.rs

```diff
@@ -245,28 +246,53 @@ where
     async fn handle_event(&mut self, event: BspConfirmedStoring) -> anyhow::Result<()> {
         info!(
             target: LOG_TARGET,
-            "Runtime confirmed BSP storing file: {:?}",
-            event.file_key,
+            "Runtime confirmed BSP storing files: {:?}",
+            event.file_keys,
         );
-
-        let file_key: HasherOutT<T> = TryFrom::<[u8; 32]>::try_from(*event.file_key.as_ref())
-            .map_err(|_| anyhow::anyhow!("File key and HasherOutT mismatch!"))?;
+        let mut successful_metadatas = Vec::new();
 
         // Get the metadata of the stored file.
         let read_file_storage = self.storage_hub_handler.file_storage.read().await;
-        let file_metadata = read_file_storage
-            .get_metadata(&file_key)
-            .expect("Failed to get metadata.");
+
+        for fk in &event.file_keys {
+            match TryFrom::<[u8; 32]>::try_from(*fk.as_ref()) {
```

Comment:
Can you add a TODO to use the `convert_raw_bytes_to_hasher_out` method from `rocksdb.rs`? We can move it to  `shc_common` as it's frequently needed.